### PR TITLE
Recommendation product `description` is displayed instead of `id`

### PR DESCRIPTION
**Pull request by ruivieira**
(see #12) @elmiko ptal

### CODE REVIEW

**Comment by elmiko** [comment]
it looks good for the first couple users, but it seems some movie titles are not getting display properly. to replicate, rebase this on top of master and try getting the titles for user 100

i see this
image

**Comment by elmiko** [comment]
this bug is being difficult to reproduce, we'll merge for now and continue to chase it.

**Comment by sophwats** [comment]
It seems that issue #21 is seeing this problem with films not being displayed correctly.

File: src/main/java/io/radanalytics/jiminy/PredictorUtils.java [hunk 1]
```diff
@@ -55,7 +52,7 @@ public static ReportDAO fetchPredictions(String uid, int topk, String predictorU
 			RankType rank = mapper.readValue(output, RankType.class);
 			 
 			report = new ReportDAO("OK",
-					Arrays.asList(new ColumnHeader("id", "ID"), new ColumnHeader("rating", "Rating")),
+					Arrays.asList(new ColumnHeader("description", "Description"), new ColumnHeader("rating", "Rating")),
```
Comment (sophwats) [inline src/main/java/io/radanalytics/jiminy/PredictorUtils.java]: ?is this ColumnHeader where issue 21 is arising? @elmiko @ruivieira

**Pull request merged**